concent_api/core/utils.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+import math
+
+from django.conf                    import settings
+
+from golem_messages                 import constants
+from golem_messages                 import message
+from golem_messages.helpers         import maximum_download_time
+from golem_messages.helpers         import subtask_verification_time
+
+
+def calculate_maximum_download_time(size: int, rate: int = constants.DEFAULT_UPLOAD_RATE) -> int:
```

Comment:
I glossed over it earlier, but now I think we should remove this default. We have a setting for it (`MINIMUM_UPLOAD_RATE`) and it's too easy to forget to set it.

Reply:
:+1: